src/fsrs/models.py

```diff
@@ -262,24 +262,27 @@ def from_dict(source_dict: dict[str, Any]) -> "Card":
             last_review,
         )
 
-    def get_retrievability(self, now: datetime) -> Optional[float]:
+    def get_retrievability(self, now: Optional[datetime] = None) -> float:
         """
         Calculates the Card object's current retrievability for a given date and time.
 
         Args:
             now (datetime): The current date and time
 
         Returns:
-            Optional[float]: The retrievability of the Card object if it's in the Review state, otherwise, will return None.
+            float: The retrievability of the Card object.
         """
         DECAY = -0.5
         FACTOR = 0.9 ** (1 / DECAY) - 1
 
-        if self.state == State.Review:
+        if now is None:
+            now = datetime.now(timezone.utc)
+
+        if self.state in (State.Learning, State.Review, State.Relearning):
             elapsed_days = max(0, (now - self.last_review).days)
             return (1 + FACTOR * elapsed_days / self.stability) ** DECAY
```

Comment:
Setting `self.stability = 0` will trigger an error.

Reply:
self.stability = 0 only when the state is new, right?

Reply:
![image](https://github.com/user-attachments/assets/1f2fcb00-dfa8-4aed-8d16-ec2cc4a8100e)
Continuously selecting "Again" will cause the stability to approach zero indefinitely.

Reply:
Good point. I haven't known that.

Reply:
@ishiko732 Yeah, explicitly setting a card's stability to 0 would cause an error if the card is not in the New state. This would probably have to be done by the user though, which I'm not sure they would do. Stability is supposed to be computed internally by the code, not set by user.

Reply:
@L-M-Sherlock You're asking if stability asymptotically approaches 0 when a card's rated again a bunch of times? Yeah it does
![Cursor_and_deleteme](https://github.com/user-attachments/assets/43449edc-1ec7-4734-a5a0-445f0c31b915)